### ISSUE TITLE
[save] .vds is the VisiData internal save format

### ISSUE THREAD

**Issue by kalyan860** [saulpw/visidata#2175]
I often create small tables in visidata, entering the data myself manually. I do it in visidata instead of excel as it's quicker. And then I run a few short analysis. Now I want to save this data for future use. And the saved data should retain column types, derived column formulas etc. When I save to tsv, csv or excel, these are gone. And I have to  re-edit the excel file to convert columns to correct types etc. 

I am aware of saving command log through Ctrl-D and replaying it. But it doesn't work as intended for my use case. Once it kept replaying all the Ctrl-S I did endlessly. And saving to .vd is causing data loss for some reason. 

In short, visidata should have its own native filetype to save like sc-im had .sc.

**Comment by aborruso** [comment]
Hi @kalyan860 there is the `vds` format and I think it retains the fields format

**Comment by anjakefala** [comment]
Yes, it does!

You can set it as your default with `options.save_filetype='vds'` in your `~/.visidatarc`.

> And saving to .vd is causing data loss for some reason

If you do have a small reproduction, I would love it if you could share what you are seeing.

**Comment by saulpw** [comment]
Yes, @anjakefala is correct, `.vds` is the VisiData "internal" format.  It's very simple and I'm sure there are lots of ways it doesn't work (@reagle pointed out today that saving multiple sheets into .vds saves also the index sheet, which is not right), but we're heading in that direction, so if you discover specific issues with saving to .vds, please file Github issues and we'll address them.
